Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -658,6 +659,33 @@ def test_updown_arrow_with_completion_menu(self):
         # so we should end up where we were when we initiated tab completion.
         output = multiline_input(reader, namespace)
         self.assertEqual(output, "os.")
+        # TODO: The menu should be visible, but currently isn't?
```

Comment:
Is this TODO still relevant?

Reply:
Essentially, I wanted to assert stuff while the tab completions menu is displayed, but I failed to figure out how to do it. I suspect the `multiline_input` function is not a good vehicle for that.

It would be nice to have a test that asserts that the menu is indeed displayed below and that the cursor behaves in a sane way when user presses various arrow keys while the menu is displayed. But I will not have time to figure it out in the next week I am afraid.

For now I've removed the TODO and the test below.